File: app/src/main/java/com/vkpapps/thunder/model/AudioInfo.java
package com.vkpapps.thunder.model;

/***
 * @author VIJAY PATIDAR
 * */
public class AudioInfo {

    private String path;
    private String name;
    private boolean selected;


    public AudioInfo() {

    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/vkpapps/thunder/model/AudioInfo.kt
package com.vkpapps.thunder.model

import android.net.Uri
import androidx.annotation.NonNull
import androidx.room.Entity
import androidx.room.PrimaryKey

/***
 * @author VIJAY PATIDAR
 */
@Entity
class AudioInfo(var uri: Uri, var name: String) {
    @PrimaryKey
    @NonNull
    lateinit var id: String
    var isSelected = false
}